wagtail/admin/templates/wagtailadmin/edit_handlers/inline_panel_child.html

```diff
@@ -1,5 +1,14 @@
 {% load i18n %}
 <li data-inline-panel-child id="inline_child_{{ child.form.prefix }}">
+    {% if child.form.non_field_errors %}
+    <ul>
+        <li class="error-message">
+            {% for error in child.form.non_field_errors %}
```

Comment:
Shouldn't the for loop include the `<li>` and `</li>`? This would make better sense semantically if it did.

Reply:
Makes sense, implemented it!